config/browser-extension/ext.dev.config.js

```diff
@@ -0,0 +1,27 @@
+const merge = require('webpack-merge');
+const CreateFileWebpack = require('create-file-webpack');
+
+const common = require('./ext.base.config');
+
+// An arbitrary key used for development; if we do not set it, it will be automatically generated and we would have to
+// update our publicPath in the runtime, which proves to be hard in case of a browser extension
+const KEY = 'hhjfhkdpnajfnekdaigmpahnnoccfaiz';
```

Comment:
Yes, there's been a change. Now there's no need to change the key per development instance. We set the key for the development app instead of having Chrome generate a random one for us. 

In the future, if we want to keep the development instance's identical with the production instance (e.g. when we use some external service), we can put there the production key.
I'm just looking deeper into the production issues to see if this kind of setup is also safe for production. **For development there are no downsides I know of.**